### PR TITLE
Fix/prettier format

### PR DESCRIPTION
**Pull request by oleiade**
🔴  This is based on https://github.com/grafana/k6-jslib-utils/pull/9

As I was implementing https://github.com/grafana/k6-jslib-utils/pull/9 I realized we had a mixed formatting in the codebase. This PR adds prettier configuration, and a pre-commit hook that will attempt to format staged files automatically.

This is really a tentative PR as I'm uncertain if we would prefer prettier to ESLint, but it was just the one VSCode made me use as a default and I stuck to it. Please let me know if you're open to this change, or we should do this in any other way 🙏🏻

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,35 @@
...
 
 Download the latest release from https://jslib.k6.io/
 
-
 ## Example
 
 ```javascript
-import { sleep } from 'k6';
-import http from 'k6/http';
+import { sleep } from 'k6'
```
Comment (codebien) [inline README.md:12]: This makes me think we are not using the same configuration as we have for https://github.com/grafana/k6-docs examples. I guess they should be consistent.

Comment (oleiade) [reply README.md:12]: k6 docs use a mix of ESLint and Prettier as far as I could see. Also, as we don't have a unified config, should we consider k6-docs as the de-facto standard, what do you folks think?

Could be worth unifying our configuration across all jslibs for instance?

**Comment by sniku** [comment]
My apologies @oleiade for not noticing this PR for almost a year. 
I would have merged it right away, but now we have conflicts. If you still feel this should be done, please fix the conflicts and merge :-)

**Comment by oleiade** [comment]
Let's do it 👍🏻

**Pull request merged**